src/shared/ui/list/list.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { render, screen } from '@testing-library/react';
+import { List, cx } from './list';
+
+const data = ['listItem 1', 'listItem 2', 'listItem 3'];
+
+describe('List Component', () => {
+  it('renders list correctly', () => {
+    render(<List data={data} />);
+
+    const list = screen.getByTestId('list');
+
+    expect(list).toBeVisible();
+  });
+
+  it('list has marked by default', () => {
+    render(<List data={data} />);
+
+    const list = screen.getByTestId('list');
+
+    expect(list).toHaveClass(cx('marked'));
+  });
+
+  it('renders list correctly if list-data is empty', () => {
+    render(<List data={[]} />);
+
+    const list = screen.getByTestId('list');
+
+    expect(list).toBeInTheDocument();
+  });
+
+  it('renders list-items correctly', () => {
+    render(<List data={data} />);
+
+    const items = screen.getAllByTestId('list-item');
+
+    items?.forEach((item) => {
+      expect(item).toBeVisible();
+    });
+  });
```

Comment:
It doesn't check the content inside

Reply:
Done

Reply:
Just FYI: you could use `expect(item).toHaveTextContent(data[i]);` instead of `expect(item.innerHTML).toBe(data[i]);
`